tracing-serde/README.md

```diff
@@ -55,7 +55,7 @@ extern crate tracing_serde;
 use tracing_serde::AsSerde;
 ```
 
-Please read the [`tracing` documentation](https://docs.rs/tracing/0.1.0/tracing/index.html)
+Please read the [`tracing` documentation](https://docs.rs/tracing/0.1.1/tracing/index.html)
```

Comment:
this looks like an overeager find-and-replace? I think we should just make links to other crates point to `latest`; it's a pain keeping these up to date
```suggestion
Please read the [`tracing` documentation](https://docs.rs/tracing/latest/tracing/index.html)
```